tests/e2e/test_e2e_docker.py

```
@@ -0,0 +1,110 @@
+        _IMAGE,
+        "--persona", "/config/persona.json", "--host", "0.0.0.0", "--port", "8337",
+        timeout=60,
+    )
+
+    url = f"http://127.0.0.1:{_PORT}"
+    deadline = time.time() + 60
+    while time.time() < deadline:
+        try:
+            if httpx.get(f"{url}/api/tags", timeout=2).status_code == 200:
+                break
+        except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)